_integration-schemas/3plcentral/foreign-keys.md

```diff
@@ -0,0 +1,139 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas-foreign-keys/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+tap-reference: "3plcentral"
+
+version: "1.0"
+
+foreign-keys:
+  - id: "customer-id"
+    table: "customers"
+    attribute: "customer_id"
+    all-foreign-keys:
+      - table: "customers"
+        join-on: "customer_id"
+      - table: "sku_items"
+        join-on: "customer_id"
+      - table: "stock_details"
+        join-on: "customer_id" 
+      - table: "inventory"
+        join-on: "customer_identifier" 
+      - table: "orders"
+        join-on: "customer_identifier"
```

Comment:
I think this and `subattribute` (also, not `subattributes` - singular here) were flipped. It should look like this:

```
- table: "orders"
  subattribute: "customer_identifier"  ## Name of the subattribute (object or array) 
  join-on: "id"  ## Name of the actual field that's the foreign key
```

It looks like a bunch of these got flipped in this file. I'm not gonna leave a comment on them all lest it be overwhelming, but I'd go through this and switch things around.